src/crud_operations.py

```diff
@@ -69,8 +71,8 @@ def set_message(session: Session, message: Message) -> None:
         "msg_author_id, msg_author_first_name, "
         "msg_author_last_name, msg_author_username, msg_text) VALUES ("
         f"'{chat_id}',{msg_id},'{msg_link}','{msg_date}','{msg_time}',{msg_author_id},"
-        f"'{msg_author_first_name}','{msg_author_last_name}','{msg_author_username}',"
-        f"'{msg_text}')"
+        f"'{json.dumps(msg_author_first_name)}','{json.dumps(msg_author_last_name)}','{json.dumps(msg_author_username)}',"
```

Comment:
В связи с изменениями строка стала слишком длинной. Давай разобьём её:
```suggestion
        f"'{json.dumps(msg_author_first_name)}',"
        f"'{json.dumps(msg_author_last_name)}',"
        f"'{json.dumps(msg_author_username)}',"
```